integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/predicate/MatchSearchPredicateIT.java

```diff
@@ -781,18 +781,21 @@ public void analyzerOverride() {
 
 		String whitespaceAnalyzedField = indexMapping.whitespaceAnalyzedField.relativeFieldName;
 		String whitespaceLowercaseAnalyzedField = indexMapping.whitespaceLowercaseAnalyzedField.relativeFieldName;
+		String whitespaceLowercaseSearchAnalyzedField = indexMapping.whitespaceLowercaseSearchAnalyzedField.relativeFieldName;
 
 		SearchQuery<DocumentReference> query = scope.query()
 				.predicate( f -> f.match().field( whitespaceAnalyzedField ).matching( "NEW WORLD" ) )
 				.toQuery();
 
+		// Terms are never lower-cased, nether at write nor at query time.
```

Comment:
Maybe these comments would be easier to understand if they were above the query they are referring to? I was confused for a minute. You could remove the blank line between the query and the `assertThat` to make it clearer that they are related.